### PR TITLE
Added ELEMENT_EXISTS function

### PR DESCRIPTION
**Pull request by ziflex**

### CODE REVIEW

File: e2e/main.go [hunk 1]
```diff
@@ -48,6 +54,19 @@ func main() {
 		Dir:  filepath.Join(*pagesDir, "dynamic"),
 	})
 
+	var filterR *regexp.Regexp
+
+	if *filter != "" {
+		r, err := regexp.Compile(*filter)
+
+		if err != nil {
```
Comment (3timeslazy) [inline e2e/main.go]: I think would be better to use ```regexp.MustCompile(*filter)``` instead.

File: e2e/main.go [hunk 2]
```diff
@@ -3,12 +3,12 @@ package main
 import (
 	"flag"
 	"fmt"
-	"os"
-	"path/filepath"
-
 	"github.com/MontFerret/ferret/e2e/runner"
 	"github.com/MontFerret/ferret/e2e/server"
 	"github.com/rs/zerolog"
+	"os"
```
Comment (3timeslazy) [inline e2e/main.go]: Here you can use ```goimports``` util.

File: pkg/drivers/cdp/element.go [hunk 3]
```diff
@@ -679,6 +679,33 @@ func (el *HTMLElement) CountBySelector(selector values.String) values.Int {
...
+			Msg("failed to retrieve nodes by selector")
+
+		return values.False
+	}
+
+	if res.NodeID == 0 {
+		return values.False
+	}
+
+	return values.True
```
Comment (3timeslazy) [inline pkg/drivers/cdp/element.go]: What about ```return values.NewBoolean(res.NodeID == 0)```?

File: pkg/drivers/http/element.go [hunk 4]
```diff
@@ -248,6 +248,16 @@ func (el *HTMLElement) CountBySelector(selector values.String) values.Int {
...
 }
 
+func (el *HTMLElement) ExistsBySelector(selector values.String) values.Boolean {
+	selection := el.selection.Closest(selector.String())
+
+	if selection == nil {
+		return values.False
+	}
+
+	return values.True
```
Comment (3timeslazy) [inline pkg/drivers/http/element.go]: The same with [this](https://github.com/MontFerret/ferret/pull/210/files#r242466319).

File: e2e/main.go (hunk 1 above)
Comment (ziflex) [reply e2e/main.go]: That would panic the app without a proper message.

File: e2e/main.go (hunk 2 above)
Comment (ziflex) [reply e2e/main.go]: @3timeslazy Could you give me more details on that?

File: pkg/drivers/cdp/element.go (hunk 3 above)
Comment (ziflex) [reply pkg/drivers/cdp/element.go]: @3timeslazy what about code readability? :)

File: e2e/main.go (hunk 2 above)
Comment (3timeslazy) [reply e2e/main.go]: https://github.com/bradfitz/goimports

File: e2e/main.go (hunk 1 above)
Comment (3timeslazy) [reply e2e/main.go]: It will.

Comment (3timeslazy) [reply e2e/main.go]: It will.

File: pkg/drivers/cdp/element.go (hunk 3 above)
Comment (3timeslazy) [reply pkg/drivers/cdp/element.go]: ¯\_(ツ)_/¯

**Pull request merged**